### ISSUE TITLE
[Bug]: Missing languages?

### ISSUE THREAD

**Issue by tomtomme** [jasp-stats/jasp-issues#2836]
### JASP Version

0.19.0

### Bug Description

![grafik](https://github.com/user-attachments/assets/68a90a9a-a538-46f2-8b0d-a283e3827150)

Indonesian is more complete than e.g. french (see below), so it should not be marked that way anymore.
Also if we add italian and russion we should probably add many more languages too, see:

![grafik](https://github.com/user-attachments/assets/4dbc78a4-6d08-4428-97b6-dd11daecc33f)

### Expected Behaviour

Add more langs?

**Comment by tomtomme** [comment]
ah I see. Thx for the explainer.
